src/main/java/org/folio/circulation/domain/OverduePeriodCalculatorService.java

```diff
@@ -89,17 +97,20 @@ private int getMinutesOfDay(LocalTime time) {
   Result<Integer> adjustOverdueWithGracePeriod(Loan loan, int overdueMinutes) {
     int result = shouldIgnoreGracePeriod(loan)
       ? overdueMinutes
-      : Math.max(overdueMinutes - getGracePeriodMinutes(loan), ZERO_MINUTES);
+      : overdueMinutes > getGracePeriodMinutes(loan) ? overdueMinutes : ZERO_MINUTES;
```

Comment:
I find it difficult to read two ternary operations chained together, maybe a method could be extracted to improve the clarity of the second case?